CAIPs/caip-25.md

```diff
@@ -108,6 +112,8 @@ If a connection is rejected, the wallet MAY respond with a generic error or sile
 }
 ```
 
+The `wallet` object MUST be included for wallet-level method and notification requests.
```

Comment:
I'm not totally clear what this means. Must it be included if there are no `wallet-level` methods or notifications?